app/views/TaxYearSummary.scala.html

```diff
@@ -191,12 +191,12 @@ <h2 class="govuk-heading-m">@messages(header)</h2>
 }
 
 @paymentsContent = {
-    @h2(msg = messages("tax-year-summary.payments"), classes = "govuk-heading-l")
+    @h2(msg = messages("tax-year-summary.charges"), classes = "govuk-heading-l")
     @if(viewModel.charges.isEmpty){
         @p()(content = Html(messages("tax-year-summary.payments.no-payments")))
     } else {
         <table class="govuk-table" id="payments-table">
-            <caption class="govuk-table__caption govuk-visually-hidden" style="margin-bottom: 0px;">@messages("tax-year-summary.payments")</caption>
+            <caption class="govuk-table__caption govuk-visually-hidden" style="margin-bottom: 0px;">@messages("tax-year-summary.TEST")</caption>
```

Comment:
not sure if this should refer to _tax-year-summary.TEST_ ?

Reply:
ah, that's my bad. Will make those changes